deploy/rbd/helm/README.md

```diff
@@ -4,10 +4,10 @@ The ceph-csi-rbd chart adds rbd volume support to your cluster.
 
 ## Install Chart
 
-To install the Chart into your Kubernetes cluster :
+To install the Chart into your Kubernetes cluster
 
 ```bash
-helm install --namespace "ceph-csi-rbd" --name "ceph-csi-rbd" ceph-csi/ceph-csi-rbd
+helm install --name "ceph-csi-rbd" ceph-csi/ceph-csi-rbd
```

Comment:
Why remove the namespace? This puts it in "default"

Reply:
current templates in kubernetes folder deploy everything in the default namespace, to keep consistency between both I deleted  helm deployment in a different namespace

Reply:
I don't think installing it in "default" is a good recommendation for the users. Please undo.

Reply:
undo done